src/main/java/ddingdong/ddingdongBE/domain/form/entity/Form.java

```diff
@@ -80,6 +95,31 @@ public boolean isEqualsById(Long formId) {
         return this.id.equals(formId);
     }
 
+    public void updateFormFields(List<FormField> updatedFormFields) {
+        // 삭제될 폼 필드
+        List<FormField> deletedFormFields = this.formFields.stream()
+                .filter(formField -> updatedFormFields.stream()
+                        .filter(updatedFormField -> updatedFormField.getId() != null)
```

Comment:
```suggestion
                        .filter(updatedFormField -> updatedFormField.existId())
```
p5)
이것도 사소하지만, formField 내부에서 처리해도 좋을 것 같아요!